### ISSUE TITLE
CubeCubeCube: Levels are not playable

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#10184]
### Describe the bug

Link to SWF: http://i.notdoppler.com/files/cubecubecube.swf

Game must be run in the Ruffle desktop app with `--load-behavior=blocking` option.

The menu appears fine, but in every level, the gameplay area is blank:
![image](https://user-images.githubusercontent.com/71368227/226057886-52298e54-9631-4c0a-953e-22b616b1bed1.png)

These errors are repeated in the terminal:
`ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::activation: ReferenceError: Error #1056: Cannot create property visible on flash.display.Stage3D.`
`ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::activation: ReferenceError: Error #1069: Property recompose not found on flash.geom.Matrix3D and there is no default value.`

### Expected behavior

Levels should be fully displayed and interactive.

### Affected platform

Desktop app

### Operating system

Windows 10

**Comment by Lord-McSweeney** [comment]
Alternatively, the SWF can be run in the browser, after opening the context menu and clicking "Play".

**Comment by n0samu** [comment]
With some of the upcoming PRs, now this error appears:
```
ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "enterFrame", class: flash.events::Event, ptr: 0x19f97621d98 }) to handler FunctionObject(FunctionObject { ptr: 0x19f8e00b6d8 }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: recoverFromDisposal)
        at away3d.containers::View3D/render()
        at cubecubecubearmorversion_fla::MainTimeline/onEnterFrame()
```

**Comment by Lord-McSweeney** [comment]
Those errors are caused by this error: `ReferenceError: Error #1056: Cannot create property visible on flash.display.Stage3D.`

**Comment by n0samu** [comment]
Fixed by #11200
